crates/liboci-cli/src/exec.rs

```diff
@@ -49,3 +76,19 @@ where
         .ok_or_else(|| format!("invalid KEY=value: no `=` found in `{s}`"))?;
     Ok((s[..pos].parse()?, s[pos + 1..].parse()?))
 }
+
+fn parse_colon_separated_pair<T, U>(
```

Comment:
This is to parse the user string. I would suggest to rename this function to parse user or something along the line. Then have a comment explaining the format of the user and why we parse it as colon separated pairs.

Reply:
> This is to parse the user string. I would suggest to rename this function to parse user or something along the line. Then have a comment explaining the format of the user and why we parse it as colon separated pairs.

This is easily done, but this seems to be inconsistent with existing usage: `parse_key_val` describes the parsing format (key/value pair), not what it is used for (environment variables). I suspect we might see other use cases for a colon-separated pair (e.g. for `docker` or `podman`, the `-v` option takes such a colon-separated pair).

I'll submit this as a separate commit on the same branch, also changing the `parse_key_val` to `parse_env`. You decide whether you want to pick that commit or not.

Reply:
@yihuaf Implemented in ec24d0a

Reply:
I see. You raised a good point. Thank you for doing this. We can revert to `parse_key_val` and `parse_colon_separated_pair`. Other than the clippy, LGTM.